src/OpenRiaServices.Tools/Test/CreateRiaClientFilesTaskTests.cs

```diff
@@ -283,6 +284,51 @@ public void CreateRiaClientFiles_Missing_Pdb_Warns()
             }
         }
 
+#if !NETFRAMEWORK
+        [TestMethod]
+        public void CreateRiaClientFiles_For_Net_60()
```

Comment:
We need to verify what this project really tests. 
I think a better approch is to create a ServerClassLibNet6.csproj or similar and use that 

The test itself should run under both NETFRAMEWORK and NET6 (so no #if)